### DISCUSSION TITLE
Thanks mate!

### DISCUSSION THREAD

**Discussion by ghost** [DarkThemeHub/GithubDarkTheme#210] [General]
Well, today I log into github and there it is - native dark mode!

They took their sweet time, but they got here in the end. Just wanted to say a quick thanks for all the fish :)

**Comment by Snazzie** [comment]
No problem! Personally not a fan of how dark it is, so i won't be using it. Which means this theme will continue to exist along side 🙂

**Comment by ghost** [comment]
> No problem! Personally not a fan of how dark it is, so i won't be using it. Which means this theme will continue to exist along side 🙂

Yeh and the 'blue on blue on blue-grey' thing... Oof. So many shades of blue. It's a classic fashion faux-pas. Stoked to hear you're sticking around!!

Perhaps you could leverage this new theme as the underlying theme for your work :)

**Reply by Snazzie** [reply]
Yeah i dont understand why they didnt use their mobile's colour scheme, it's fairly alright.

Definitely, this theme has actually been slowly reducing in size due to their new practice of using variables in `:root`.  Over the next year, i can expect this theme file to halve in size, depending on how much breaks due to them updating the rest of the elements with variables. also means less things will break every time github updates the website.

Also this discussion feature they brought out is pretty cool :P 

Maybe can explore creating a new project that is based on purely overriding through variables, then allow everyone to share "colour scheme file's" in discussion page. We'll see how much effort i have. 😅
